cartoframes/data/observatory/catalog/dataset.py

```diff
@@ -236,13 +384,29 @@ def _join_geographies_geodataframes(geographies_gdf1, geographies_gdf2):
         return join_gdf['id'].unique()
 
     def subscribe(self, credentials=None):
-        """Subscribe to a Dataset.
+        """Subscribe to a Dataset. You need Data Observatory enabled in your CARTO account.
```

Comment:
Again, I think `Dataset` should be lowercase, cause it actually refers to the `CatalogDataset`